vendor/k8s.io/kubernetes/pkg/volume/vsphere_volume/vsphere_volume.go

```diff
@@ -135,6 +135,18 @@ func (plugin *vsphereVolumePlugin) getCloudProvider() (*vsphere.VSphere, error)
 	return vs, nil
 }
 
+func (plugin *vsphereVolumePlugin) ConstructVolumeSpec(volumeName, mountPath string) (*volume.Spec, error) {
```

Comment:
should we pick up the latest fix 
https://github.com/kubernetes/kubernetes/pull/37167